pkg/controller.v1/tensorflow/job.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
How about using `corev1` to replace `v1`.

Reply:
Done.